harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -274,6 +274,17 @@ Message: |
 
 
 
+Lint:    Miscellaneous (0 priority)
+Message: |
+     138 | finding it very nice, (it had, in fact, a sort of mixed flavour of cherry-tart,
+         |                                                         ^~~~~~~ Consider adding an article before this noun.
```

Comment:
You don't want to add a/an before mass nouns. "A stuff", "an information", "a knowledge", etc.
But you must also keep in mind that most mass nouns also have countable senses: "a beer", "an area", etc. These can now all be fully queried.